projects/1.16/assets/scp-lockdown/scplockdown/lang/zh_cn.json

```
@@ -0,0 +1,248 @@
+  "block.scplockdown.straight_pipe_light_blue": "淡蓝色管道",
+  "block.scplockdown.straight_pipe_lime": "鲜绿色管道",
+  "block.scplockdown.straight_pipe_magenta": "品红色管道",
+  "block.scplockdown.straight_pipe_orange": "橙色管道",
+  "block.scplockdown.straight_pipe_pink": "粉红色管道",
+  "block.scplockdown.straight_pipe_purple": "紫色管道",
+  "block.scplockdown.straight_pipe_red": "红色管道",
+  "block.scplockdown.straight_pipe_silver": "银色管道",
+  "block.scplockdown.straight_pipe_white": "白色管道",
+  "block.scplockdown.straight_pipe_yellow": "黄色管道",
+  "block.scplockdown.sublevel_slab": "横纹半砖",
+  "block.scplockdown.sublevel_stairs": "横纹楼梯",
+  "block.scplockdown.sublevel_wall_a": "横纹墙 A",
+  "block.scplockdown.sublevel_wall_b": "横纹墙 B",
+  "block.scplockdown.sublevel_wall_c": "横纹墙 C",
+  "block.scplockdown.table": "桌子",
+  "block.scplockdown.tjunc_pipe_black": "T型黑色管道",
+  "block.scplockdown.tjunc_pipe_blue": "T型蓝色管道",
+  "block.scplockdown.tjunc_pipe_brown": "T型棕色管道",
+  "block.scplockdown.tjunc_pipe_cyan": "T型青色管道",
+  "block.scplockdown.tjunc_pipe_gray": "T型灰色管道",
+  "block.scplockdown.tjunc_pipe_green": "T型绿色管道",
+  "block.scplockdown.tjunc_pipe_light_blue": "T型淡蓝色管道",
+  "block.scplockdown.tjunc_pipe_lime": "T型鲜绿色管道",
+  "block.scplockdown.tjunc_pipe_magenta": "T型品红色管道",
+  "block.scplockdown.tjunc_pipe_orange": "T型橙色管道",
+  "block.scplockdown.tjunc_pipe_pink": "T型粉红色管道",
+  "block.scplockdown.tjunc_pipe_purple": "T型紫色管道",
+  "block.scplockdown.tjunc_pipe_red": "T型红色管道",
+  "block.scplockdown.tjunc_pipe_silver": "T型银色管道",
+  "block.scplockdown.tjunc_pipe_white": "T型白色管道",
+  "block.scplockdown.tjunc_pipe_yellow": "T型黄色管道",
+  "block.scplockdown.toilet": "马桶",
+  "block.scplockdown.vent": "通风口",
+  "block.scplockdown.ventilation": "通风管道",
+  "block.scplockdown.wall_a": "A型墙",
+  "block.scplockdown.wall_b": "B型墙",
+  "block.scplockdown.wall_c": "C型墙",
+  "block.scplockdown.wall_d": "D型墙",
+  "block.scplockdown.wall_e": "E型墙",
+  "block.scplockdown.wall_f": "F型墙",
+  "block.scplockdown.wall_g": "G型墙",
+  "block.scplockdown.wall_slab": "A型半砖",
+  "block.scplockdown.wall_stairs": "A型楼梯",
+  "block.scplockdown.white_slab": "白色半砖",
+  "block.scplockdown.white_stairs": "白色楼梯",
+  "block.scplockdown.white_wall": "白墙",
+  "block.scplockdown.wood_crate": "木箱",
+  "container.crate": "周转箱",
+  "container.locker": "储物柜",
+  "item.scplockdown.keycard_level1": "一级钥匙卡",
+  "item.scplockdown.keycard_level2": "二级钥匙卡",
+  "item.scplockdown.keycard_level3": "三级钥匙卡",
+  "item.scplockdown.keycard_level4": "四级钥匙卡",
+  "item.scplockdown.keycard_level5": "五级钥匙卡",
+  "item.scplockdown.screwdriver": "改锥",
+  "itemGroup.lockdown_building_blocks": "SCP：封锁 建筑方块",
+  "itemGroup.lockdown_items": "SCP：封锁 物品",
+  "lockdown.card-reader.changelevel": "已将权限级别更变为%1$s",
+  "lockdown.card-reader.fail.required": "最低权限级别需求为：%1$s",
+  "screen.computer.display": "显示字符串",
+  "screen.computer.url": "URL",
+  "tooltip.card-reader.0": "使用改锥更变权限级别",
+  "tooltip.straight-pipe.0": "在创造模式下右键以更变类型"
```

```suggestion
  "tooltip.straight-pipe.0": "在创造模式下右击以更变类型（漏译内容）"
```
有漏译

**右键**不是一个完整的谓语，不能作为一个动作使用。
例如，`右键单击` 或 `右击` 、`右键长按` 或 `长按右键`都是可以的。但 `右键` 缺少语言成分。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。